_authors/osci_2023/Sanjana679.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Sanjana Nandi 
+short_name: Sanjana
+twitter: 
```

Comment:
Can you remove this line if you are not using it?

Reply:
Sure! I'll do that right now

Reply:
Oh, I got what you meant. I'll delete the twitter line.